encoding/decode.go

```diff
@@ -0,0 +1,48 @@
+package encoding
+
+import (
+	"encoding/base64"
+	"fmt"
+	"strings"
+
+	"github.com/go-chat-bot/bot"
+)
+
+func decode(command *bot.Cmd) (string, error) {
+
+	if len(command.Args) < 2 {
+		return invalidAmountOfParams, nil
+	}
+
+	var str string
+	var err error
+	switch command.Args[0] {
+	case "base64":
+		s := strings.Join(command.Args[1:], " ")
+		str, err = decodeBase64(s)
+	default:
+		return invalidParams, nil
+	}
+
+	if err != nil {
+		return fmt.Sprintf("Error: %s", err), nil
+	}
+
+	return str, nil
+}
+
+func decodeBase64(str string) (string, error) {
+	data, err := base64.StdEncoding.DecodeString(str)
+	if err != nil {
+		return "", err
+	}
+	return fmt.Sprintf("%s", data), nil
```

Comment:
You can use just:  `return string(data), nil` there's no need to fmt a []byte just to converto to string ;)